lib/fileutils.rb

```diff
@@ -831,9 +833,18 @@ def copy_entry(src, dest, preserve = false, dereference_root = false, remove_des
   end
   module_function :copy_entry
 
+  # Copies file from +src+ to +dest+, which may not be directories:
```

Comment:
Let's use `must not` or `should not` instead of `may not` here.

Reply:
Fixed.